### ISSUE TITLE
Transform paths doesn't work with import require statements

### ISSUE THREAD

**Issue by richardspence** [LeDDGroup/typescript-transform-paths#26]
I tried out this plugin and it works great for normal es6 import statements with default or named exports.  Unfortunately it doesn't work on files with the legacy commonJS export shape.

A workaround is to change the import to es6 and use something like the `esModuleInterop` compiler flag.  That requires modification of the source code though, and therefore is a heavy handed.

## Dependency.ts
```ts
export = 3;
```
## Main.ts
```ts
import d = require('Dependency');
```

## Expected Main.d.ts
```ts
import d = require('./Dependency');
```

## Actual
No transform was performed

**Comment by danielpza** [comment]
Hi @richardspence , thanks for reporting the issue, [a PR](#27) is up and should merged and publish in short.

@all-contributors please add @richardspence for bug.

**Comment by danielpza** [comment]
@richardspence, a fix is up in *v.1.1.7*, please upgrade

**Comment by richardspence** [comment]
@danielpza the fix doesn't quite handle the scenario I mentioned.  The code/tests are looking for `var x = require('...')` where it needs to instead look at `import x = require('...');`

**Comment by danielpza** [comment]
@richardspence, I made a PR with a fix #29, it should cover your use case.

> I'd also avoid looking at vanilla require statements and instead limit the scope to top level imports only

Could you explain why?

**Comment by richardspence** [comment]
Someone could have a locally scoped require variable and this could introduce unintended consequences. I think it's ok to be opinionated that this only supports the import syntax (and the constraints that come with it (like must be top level, etc.).

There are other edge cases like require invocations can be dynamic and pass variables or concatenations - and that's all avoided by restricting yourself to `import` only.

**Comment by danielpza** [comment]
I see, while I guess it would still apply to commonjs/require modules, it could be written with `import`, so I think we won't miss it there. I'll rollback those changes, thanks :+1:

**Comment by danielpza** [comment]
@richardspence, a fix is up, upgrade to *v1.1.8*
